packages/web-runtime/src/store/spaces.ts

```diff
@@ -69,6 +74,17 @@ const actions = {
 
     const spaces = graphResponse.data.value.map((space) => buildSpace(space))
     context.commit('LOAD_SPACES', spaces)
+  },
+  async loadSpaceQuotas(context, { httpAuthenticatedClient }) {
+    // FIXME: Use graphClient after added to libre-graph-api specs https://github.com/owncloud/libre-graph-api
+    const { data } = await httpAuthenticatedClient.get(
+      `${context.rootGetters.configuration.server}graph/v1.0/drivequotas`
+    )
+    context.commit('LOAD_SPACE_QUOTAS', {
+      maxProjectQuota: data.max_project_quota || 0,
```

Comment:
I don't understand the API decision. Why is it not a map by drive type?

Reply:
We could even include a max quota in the individual drives, without the need for an additional endpoint.

Reply:
What do you mean with map by drive type?
project_quota and personal_quota are mapped by the naming already

Reply:
I also don’t know if this is future proof at all or just for the experimental branch, I think @butonic will have some objections against the endpoint as it is not defined in the ms graph

Reply:
I mean to **only** use the drive type. The endpoint is `drivequotas` and the state prop is also `quotas`. So IMO it would be sufficient to use the drive types, e.g. `personal`, `project` as keys. Already backend side, so that you wouldn't have to map the quota types manually into your own props.

Reply:
As I said, it could just be another prop in each drive.

Reply:
So I see there are two problems here:

1) The format of the response
I see you would prefer a map driveType to maxsize that makes sense. Allthough the endpoint could no longer report default quotas. But that is fine for me.

2) The location of the response
I agree that adding an endpoint that is not in the MS Graph spec doesn't make much sense. But the drive also has no prop for max quota, so that would be a custom change too. As far as I see there is no concept of a max quota in graph. Additionally, if we add it to the drive, a user with no drive will not be able to see this value.

Reply:
@kulmann @kobergj 
I would highly suggest using the capabilities provider for that, we already have a spaces property, where the (max) quota configuration could be integrated

Reply:
Unfortunately that is not possible. The capabilities live in reva. Graph is an ocis feature. We cannot get the config value from graph to the capabilities endpoint. The same value would have to be configured in both reva and graph. This is bad. The reva value will be outdated in 12mins.

So the only one who can answer question about max drive quota is the graph service

Reply:
we should think twice before we extend the graphapi with something that is not part of the spec...

Reply:
Yes I agree. BUT the graph service enforces the max quotas. So it is the only one that can report the value for it.

Reply:
oh no 😢 then i get it, thanks for clarifying

Reply:
@fschade @kobergj 
Maybe we can use this endpoint 
https://docs.microsoft.com/de-de/graph/api/usersettings-get?view=graph-rest-1.0

Reply:
That endpoint is supposed to return a `userSettings` object https://docs.microsoft.com/de-de/graph/api/resources/usersettings?view=graph-rest-1.0 We would also need to add a custom prop